### ISSUE TITLE
Can you support v1 version?

### ISSUE THREAD

**Issue by phonowell** [MonoEven/toml4ahk#1]
First of all, thank you very much for your work. Compared to ini, toml is really a step ahead of its time.
But I still dare to ask, is there any plan for your project to support v1 version of ahk?
After all, there are still some users (like me) who still use v1 as their main development environment.

**Comment by MonoEven** [comment]
thanks for noting the project of toml4ahk.
but I think it will be difficulty for me to support v1.
the worst thing is I reference java resources, it means class && many types (like v2 array or map) must performance well whenever in my "Gson" or "HashMap" or "ArrayList" and another things. but you can see, v1 has no clear concept of type, it make it difficulty for supporting v1.
so, I may not do this work, but the better suggestion is you are possible to use the library like v1-call-v2.(thqby https://www.autohotkey.com/boards/viewtopic.php?f=6&t=98917&hilit=v2)
once again, thank you for your interest.

**Comment by phonowell** [comment]
Thank you for your response.
v1 version does have a lot of shortcomings, and in fact, I've had a lot of inconvenient places in my own use. I improved the programming experience by using a translator that I built to translate javascript to v1 version's code, but unfortunately, it also made it hard for me to migrate to v2 version easily.
Thank you very much for your solution, I'll take a closer look. And have a great day as well.
